sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClientTest.java

```diff
@@ -3,12 +3,12 @@
 
 package com.azure.messaging.servicebus.administration;
 
+import com.azure.core.credential.AccessToken;
```

Comment:
Other than the createQueue Test, the remaining changes are for 25146 PR, please ignore the rest of the changes